### PR TITLE
migrate docusaurus v1 to v2

### PR DESCRIPTION
**Pull request by jakubjezek001**
Migrating docusaurus will probably take some conversion by hand and not everything will be possible to do by their automation command documented at here https://v2.docusaurus.io/docs/migrating-from-v1-to-v2/#migration-command

This description is work in progress

### CODE REVIEW

**Comment by jakubjezek001** [comment]
To reproduce the log bellow just do:
1. `cd [this repo folder]/website_v2`
2. `yarn install`
3. `yarn build`
4. `yarn start`

```
./docs/admin_config.md
SyntaxError: C:\CODE\pypeclub-github-io\website_v2\docs\admin_config.md: Expected corresponding JSX closing tag for <td> (67:92)

  65 | <tr parentName="tbody">
  66 | <td parentName="tr" {...{"align":null}}>{`root`}{`[\\<root name`}{`>]`}</td>
> 67 | <td parentName="tr" {...{"align":null}}>{`Path to root folder when multiple roots are used`}</br>{`Key `}<inlineCode parentName="td">{`<root name>`}</inlineCode>{` represents root key specified in `}<inlineCode parentName="td">{`roots.json`}</inlineCode></td>
     |                                                                                             ^
  68 | </tr>
  69 | <tr parentName="tbody">
  70 | <td parentName="tr" {...{"align":null}}>{`project`}{`[name]`}</td>
 @ ./.docusaurus/registry.js 1:117047-117097 1:116943-117014
 @ ./node_modules/@docusaurus/core/lib/client/exports/ComponentCreator.js
 @ ./.docusaurus/routes.js
 @ ./node_modules/@docusaurus/core/lib/client/clientEntry.js
 @ multi ./node_modules/@docusaurus/core/lib/client/clientEntry.js./docs/admin_ftrack.md
SyntaxError: C:\CODE\pypeclub-github-io\website_v2\docs\admin_ftrack.md: Unterminated JSX contents (140:16)

  138 | <h3 {...{"id":"where-to-find-api-key"}}>{`Where to find API key`}</h3>
  139 | <p>{`Please check the `}<a parentName="p" {...{"href":"http://ftrack.rtd.ftrack.com/en/backlog-scaling-ftrack-documentation-story/developing/api_keys.html"}}>{`official documentation`}</a>{`.`}</p>
> 140 |     </MDXLayout>
      |                 ^
  141 |   )
  142 | };
  143 | MDXContent.isMDXComponent = true
 @ ./.docusaurus/registry.js 1:67259-67309 1:67155-67226
 @ ./node_modules/@docusaurus/core/lib/client/exports/ComponentCreator.js
 @ ./.docusaurus/routes.js
 @ ./node_modules/@docusaurus/core/lib/client/clientEntry.js
 @ multi ./node_modules/@docusaurus/core/lib/client/clientEntry.js./docs/admin_presets_plugins.md
SyntaxError: C:\CODE\pypeclub-github-io\website_v2\docs\admin_presets_plugins.md: Expected corresponding JSX closing tag for <td> (125:98)

  123 | <tr parentName="tbody">
  124 | <td parentName="tr" {...{"align":null}}><strong parentName="td">{`sequence`}</strong></td>
> 125 | <td parentName="tr" {...{"align":null}}>{`Generate a sequence of images instead of single frame.`}</br>{`Is applied only if `}<strong parentName="td">{`"ext"`}</strong>{` of output is image extension e.g.: png or jpg/jpeg.`}</td>
      |                                                                                                   ^
  126 | </tr>
  127 | </tbody>
  128 | </table>
 @ ./.docusaurus/registry.js 1:102137-102196 1:102015-102095
 @ ./node_modules/@docusaurus/core/lib/client/exports/ComponentCreator.js
 @ ./.docusaurus/routes.js
 @ ./node_modules/@docusaurus/core/lib/client/clientEntry.js
 @ multi ./node_modules/@docusaurus/core/lib/client/clientEntry.js./docs/artist_tools.md
SyntaxError: C:\CODE\pypeclub-github-io\website_v2\docs\artist_tools.md: Expected corresponding JSX closing tag for <li> (216:90)

  214 | <li parentName="ol">{`Open Workfiles tool from pipeline menu`}</li>
  215 | <li parentName="ol">{`Select file from list - the latest version is the highest `}<em parentName="li">{`(descendent ordering)`}</em></li>
> 216 | <li parentName="ol">{`Press `}<inlineCode parentName="li">{`Open`}</inlineCode>{` button`}</div>
      |                                                                                           ^
  217 | <div class="ImagePanel.image Screenshot"></li>
  218 | </ol>
  219 | <p><img alt={"workfiles_1"} src={require("!url-loader?limit=10000&name=assets/images/[name]-[hash].[ext]&fallback=file-loader!./assets/workfiles_1.png").default}  /></p>```

**Comment by antirotor** [comment]
I've fixed those issues so site build is ok but there are still some visual issues. Those should be simple css fixes though.

Problem was in docusaurus apparently be strict with inline html syntax. We do use inline html tags in markdown documents, mainly breakline tags `<br>`. Docusaurus is treating html as xhtml (I don't know if this behaviour can be changed) and because of this standalone tags like `<br>` are invalid. You either need to specify closing tag or if there is not one then use `<br />` or `<a />`, etc. In html you can for example omit closing tag for `<li>` but in xhtml it must have `</li>`.

So to summarize: I've replaced all occurances of `<br>` with `<br />`. We also had in few places `</br>` which doesn't make sense in even in html standard.
